app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupUnlockViewModel.kt

```diff
@@ -70,10 +71,9 @@ class SetupUnlockViewModel @Inject constructor(
     }
 
     private fun handleEnableBiometricsClick() {
-        sendEvent(
-            SetupUnlockEvent.ShowBiometricsPrompt(
-                // Generate a new key in case the previous one was invalidated
-                cipher = biometricsEncryptionManager.createCipher(userId = state.userId),
```

Comment:
🗒️ Previously, we assumed `createCipher` was always successful and never threw an exception. Now that we're updating it to return null when exceptions are encountered, we're going to end up showing an error dialog and forcing the user to try again.

This also applies to [these](https://github.com/bitwarden/android/pull/3733/files#diff-5633ac2c8007036010438f407924e640e1f910c5fc7ffc3053dd13915404191aL161-L166) changes in `AccountSecurityViewModel`.

Reply:
This make sense but we don't need an internal event for that since this is not suspending.